wedlocks/webpack.config.js

```diff
@@ -10,7 +10,9 @@ const requiredVariables = ['SMTP_HOST', 'SMTP_USERNAME', 'SMTP_PASSWORD']
 requiredVariables.forEach(envVar => {
   if (!process.env[envVar]) {
     console.error(`Environment variable ${envVar} is required.`)
-    process.exit(1)
+    if (['dev', 'test'].indexOf(process.env.UNLOCK_ENV) > -1) {
```

Comment:
This makes sense, but I don't think it is directly related to this change

Reply:
That is right. I could not build the unlock image without it though :/